command/agent/config/test-fixtures/config-env-templates-no-name.hcl

```diff
@@ -0,0 +1,26 @@
+auto_auth {
```

Comment:
This file should be called `bad-config-...`, but I also have something similar in the validation PR, so feel free to drop it.